### PR TITLE
Microchip ADC driver fix

### PR DESCRIPTION
**Pull request by AdiIntel**
ADC readings with current microchip ADC driver aren't accurate. 
ADC configuration register need to be programmed with appropriate values for high & low clock time values.

### CODE REVIEW

**Comment by albertofloyd** [comment]
@jvasanth1 do help review this

**Review by jvasanth1 — APPROVED** [review]

**Review by jvasanth1 — APPROVED** [review]

**Review by albertofloyd — APPROVED** [review]

**Review by dcpleung — APPROVED** [review]

File: dts/bindings/adc/microchip,xec-adc.yaml [hunk 1]
```diff
@@ -17,5 +17,10 @@ properties:
     "#io-channel-cells":
       const: 1
 
+    clk_time:
```
Comment (mbolivar-nordic) [inline dts/bindings/adc/microchip,xec-adc.yaml:20]: Please change this to use `clk-time`, with a dash, to follow the general devicetree convention for property names.

**Review by mbolivar-nordic — DISMISSED** [review]

**Review by jvasanth1 — APPROVED** [review]

File: dts/bindings/adc/microchip,xec-adc.yaml (hunk 1 above)
Comment (AdiIntel) [reply dts/bindings/adc/microchip,xec-adc.yaml]: Updated property name.

**Comment by AdiIntel** [comment]
@cfriedt can you help with this PR approval.

**Pull request merged**